integration/tests/cook/test_cli.py

```diff
@@ -246,7 +246,7 @@ def test_wait_specify_timeout_and_interval(self):
         elapsed_time = time.time() - start_time
         self.assertEqual(1, cp.returncode, cp.stderr)
         self.assertIn('Timeout waiting', cli.decode(cp.stderr))
-        self.assertLess(elapsed_time, 15)
+        self.assertLess(elapsed_time, 30)
```

Comment:
How is the `15` or `30` seconds decided? Isn't `timeout` being passed as 1 second in this test?

Reply:
Yes, the timeout is 1 second. The interval, however, is the default of 5 seconds. I'm passing 30 just so that it's something large enough that the test won't be flaky. I considered removing the assertion altogether. At the end of the day, we just want to be sure that it's timing out, but I think being overly precise in the test is not buying us much.

Reply:
Is it possible users will be worried if `wait` takes too long to timeout? 
If so, I'd like to know why a tighter bound isn't working?
If not, I'll go ahead and merge.
Let me know.

Reply:
Changed it to 20

Reply:
It didn't answer my question above though :(

Reply:
> Is it possible users will be worried if wait takes too long to timeout?

Yes, that would certainly be a bad user experience.

Manual tests of the `wait` timeout show that it's behaving in a reasonable way:

```bash
$ cs submit sleep 60
Attempting to submit on dev0 cluster...
Job submission succeeded on dev0. Your job UUID is:
c1c2fcac-027d-4485-a427-5b9ed4c5af74

$ time cs wait --timeout 1 --interval 5 c1c2fcac-027d-4485-a427-5b9ed4c5af74                                                                                                                                                         
Will wait up to 1 second.
Waiting for the following jobs on dev0: c1c2fcac-027d-4485-a427-5b9ed4c5af74 ... 
Timeout waiting for response.

real    0m5.405s
user    0m0.352s
sys     0m0.024s
```

And even running the same test locally shows appropriate timeouts:

```bash
$ ./bin/single-test.sh test_wait_specify_timeout_and_interval
Found test_wait_specify_timeout_and_interval in tests/cook/test_cli.py:CookCliTest
nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
2017-11-15 09:20:50,323 [INFO] Using cook url http://localhost:12321
2017-11-15 09:20:50,324 [INFO] cs --url http://localhost:12321 submit "sleep 60"
2017-11-15 09:20:50,676 [INFO] cs --url http://localhost:12321 wait --timeout 1 a581b131-4bcf-4bc4-be38-7b266dfada9b
2017-11-15 09:20:56,055 [INFO] Elapsed time: 5.379439353942871
2017-11-15 09:20:56,056 [INFO] cs --url http://localhost:12321 wait --timeout 1 --interval 1 a581b131-4bcf-4bc4-be38-7b266dfada9b
2017-11-15 09:20:57,432 [INFO] Elapsed time: 1.376213550567627
test_wait_specify_timeout_and_interval (tests.cook.test_cli.CookCliTest) ... ok

----------------------------------------------------------------------
Ran 1 test in 7.124s

OK

real    0m7.527s
user    0m1.520s
sys     0m0.224s
```

So I'm not too interested in making the test have super tight bounds on Travis.

Reply:
okay, sounds good.